tensorboard/plugins/interactive_inference/utils/inference_utils.py

```diff
@@ -711,13 +711,13 @@ def loop_body(i, encoded_images, images):
         resized_image = tf.image.resize(image, thumbnail_dims)
         expanded_image = tf.expand_dims(resized_image, 0)
         images = tf.cond(
-            tf.equal(i, 0), lambda: expanded_image,
-            lambda: tf.concat([images, expanded_image], 0))
+            pred=tf.equal(i, 0), true_fn=lambda: expanded_image,
```

Comment:
The changes were made by the migration script. I'd revert the changes back but I want to know why TF migration script makes un-idiomatic changes. Is it that bad?

Reply:
The migration script migrates the arguments of any function with a changed signature to be keyword arguments, even if the function isn't called with arguments that were affected by the change.  I think it's just simpler that way since then you don't have to do as much analysis of the callsite and also the signature changes (which could be argument renames or reorders) - each migration can be done as a self-contained step rather than worrying about preserving the details of which args were called with names or without names.

I agree it would be nicer if it did produce idiomatic TF 2.0 code, but this is part of why manual review of the changes is recommended anyway - the goal is for the code it produces to *work* but it won't necessarily be idiomatic.